DEFAULT_CONFIG.json5

```diff
@@ -201,6 +201,17 @@
         /// Path to the TLS client public certificate
         client_certificate: null,
       },
+
+      /// **Experimental** compression feature.
+      /// Will compress the batches hop to hop (as opposed to end to end). May cause errors when
+      /// the batches's complexity is too high, causing the resulting compression to be bigger in
```

Comment:
This is no longer true, isn't?

Reply:
No, yes it is, it's an edge case but if the complexity of the batch is too high it may happen the end result is bigger than the MTU

Reply:
Ok, I had missed the point, yea if the compression result is bigger than the original batch size, then we can send the batch non compressed. However in that case we increment the batch size by 1 byte, and although that's an edge case it may happen that we exceed the maximum size for a batch by 1 byte.